core-services/release-controller/images-ocp-4.3.yaml

```diff
@@ -9,3 +9,15 @@ apiVersion: v1
 metadata:
   namespace: ocp
   name: "4.3-art-latest"
+---
+kind: ImageStream
+apiVersion: v1
+metadata:
+  namespace: ocp
+  name: "4.3-art-latest-s390x"
+  ---
```

Comment:
@tbielawa this line should not have any leading spaces

Reply:
:man_facepalming: